### DISCUSSION TITLE
[Feature request]: Installation Error due to class G4VTouchable \\ Makefile:165: recipe for target 'all' failed

### DISCUSSION THREAD

**Discussion by ghost** [OpenTOPAS/OpenTOPAS#38] [Feature-Request] [answered]
### Feature request

Hello! I am trying to install OpenTOPAS because I want to use Topas-nBio.

I have followed the [guidelines](https://github.com/OpenTOPAS/OpenTOPAS/blob/master/OpenTOPAS_quickStart_Debian.md) until 8.4. When I try to run 
```
sudo make -j20 install
```
I receive this error (I will put only the last few lines)
```
/home/cecilia/software/geant4/geant4-v11.2.0-install/include/Geant4/G4VTouchable.hh:39:40: note: ‘using G4VTouchable = class G4TouchableHistory’ has a previous declaration here
 using G4VTouchable = G4TouchableHistory;
                                        ^
chemistry/CMakeFiles/chemistry.dir/build.make:145: recipe for target 'chemistry/CMakeFiles/chemistry.dir/TsChemTrackingAction.cc.o' failed
make[2]: *** [chemistry/CMakeFiles/chemistry.dir/TsChemTrackingAction.cc.o] Error 1
CMakeFiles/Makefile2:552: recipe for target 'chemistry/CMakeFiles/chemistry.dir/all' failed
make[1]: *** [chemistry/CMakeFiles/chemistry.dir/all] Error 2
Makefile:165: recipe for target 'all' failed
make: *** [all] Error 2
```
Can anyone please help me how to solve this? I have exported the Geant4 installation directory
```export Geant4_DIR=$HOME/software/geant4/geant4-v11.2.0-install```

Thank you for your attention!

### TOPAS category

Outcome model

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by icmeyer** [comment, marked as answer]
Hi,
OpenTOPAS currently supports Geant4 v11.1.3, your output indicates you are trying with v11.2.0. Please change to the appropriate version by using the quick start instructions for selecting a Geant4 version

**Reply by ghost** [reply]
Hello! Thank you so much for your answer.

I just want to be extra sure. 

If I follow the [quick start instructions](https://opentopas.readthedocs.io/en/latest/getting-started/Debian.html), won't I have 2 Geant4? Since I already have Geant4 v11.2.0 beforehand.

Can I have 2 Geant4 on my computer or do I need to uninstall my Geant4 v11.2.0?

**Reply by icmeyer** [reply]
It's okay to have two Geant4 versions on your computer. Just make sure that when you are installing TOPAS (or running Geant4 independently if you are doing that) you use the path to correct version

If your use case is only TOPAS I would recommend uninstalling the other version to avoid confusion

**Reply by ghost** [reply]
It works well now! I have finished installing the OpenTOPAS

Thank you so much for your answer^^
